airflow/www/app.py

```diff
@@ -75,6 +75,8 @@ def create_app(config=None, testing=False):
 
     flask_app.config["PERMANENT_SESSION_LIFETIME"] = timedelta(minutes=settings.get_session_lifetime_config())
 
+    flask_app.config["MAX_CONTENT_LENGTH"] = conf.getfloat("webserver", "allowed_payload") * 1024 * 1024
```

Comment:
```suggestion
    flask_app.config["MAX_CONTENT_LENGTH"] = conf.getfloat("webserver", "allowed_payload_size") * 1024 * 1024
```

Reply:
commited in 3da440410c5b82f12e56cafa87af9a8647d73f88